TTHAnalysis/python/plotter/susy-interface/scan/masspoints/list_T2ttDiLep.txt

```diff
@@ -0,0 +1,184 @@
+600 : 520 : T2ttDiLep_600_520
```

Comment:
not sure if these txt files (and the acceptance root files) should go here or in a subdirectory in TTHAnalysis/data @cheidegg @gpetruc